lib/lndclient/LndClient.ts

```diff
@@ -238,7 +236,10 @@ class LndClient extends SwapClient {
         uris = getInfoResponse.getUrisList();
         version = getInfoResponse.getVersion();
         alias = getInfoResponse.getAlias();
-        if (channels.active <= 0) {
+
+        if (!this.isConnected()) {
```

Comment:
Because of `this.status` in `errors.UNAVAILABLE(this.currency, this.status).message` the error message already shows that LND is out of sync.

I agree on `isOutOfSync()` but don't think that a new error is necessary because of the reason stated above.

Reply:
Good point, I overlooked that.